tsdb/shard.go

```diff
@@ -543,7 +546,9 @@ func (s *Shard) validateSeriesAndFields(points []models.Point) ([]models.Point,
 
 	// get the shard mutex for locally defined fields
 	n = 0
+	var skipForFieldTypeConflict bool
 	for i, p := range points {
+		skipForFieldTypeConflict = false
```

Comment:
I hate this name.  @benbjohnson needs to pull out his uber naming skilz and help me!